applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -720,6 +722,147 @@ void SmallStrainUPwDiffOrderElement::CalculateOnIntegrationPoints(const Variable
     KRATOS_CATCH("")
 }
 
+void SmallStrainUPwDiffOrderElement::Calculate(const Variable<Vector>& rVariable,
+                                               Vector&                 rOutput,
+                                               const ProcessInfo&      rCurrentProcessInfo)
+{
+    rOutput = ZeroVector(this->GetGeometry().size() * this->GetGeometry().WorkingSpaceDimension() +
+                         mpPressureGeometry->size());
```

Comment:
This can be simplified a bit:
```suggestion
    rOutput = ZeroVector(GetNumberOfDOF());
```

Reply:
Done (used Gennady's suggestion though, with the `Vector(size, value)` constructor)